src/functions/options/docker.go

```diff
@@ -0,0 +1,35 @@
+package options
+
+import (
+	"Scruticode/src/constants"
+	"Scruticode/src/functions/utils"
+	"Scruticode/src/types"
+	"os"
+	"strings"
+)
+
+func Dockerfile() types.BaseResponse {
+	const DockerFilePath = "Dockerfile"
```

Comment:
I should check if the file `contains` the name `Dockerfile` since it can be a file named `dev.Dockerfile`

Reply:
Si se tiene que aclarar, está clara la función?

Reply:
Decis el nombre? de la func?

Reply:
Le cambio el nombre a `DockerFileExists`